client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -673,7 +675,7 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow, stepName } ) => {
 			handleSubmit(
 				{
 					selectedDesign: _selectedDesign,
-					selectedSiteCategory: categorization.selection,
+					selectedSiteCategory: categorization.selections?.join( ',' ),
```

Comment:
[nit] We could probably create a maintenance task to rename
- `category` → `categories`
- `selectedSiteCategory` → `selectedSiteCategories`

Reply:
Let's update it when we add the new metric 🙂